ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -133,17 +134,32 @@ def ocs_install_verification(
         constants.NOOBAA_DB_LABEL: 1,
         constants.NOOBAA_OPERATOR_POD_LABEL: 1,
         constants.NOOBAA_CORE_POD_LABEL: 1,
-        constants.MON_APP_LABEL: 3,
-        constants.CSI_CEPHFSPLUGIN_LABEL: number_of_worker_nodes,
-        constants.CSI_CEPHFSPLUGIN_PROVISIONER_LABEL: 2,
-        constants.CSI_RBDPLUGIN_LABEL: number_of_worker_nodes,
-        constants.CSI_RBDPLUGIN_PROVISIONER_LABEL: 2,
-        constants.OSD_APP_LABEL: osd_count,
-        constants.MGR_APP_LABEL: 1,
-        constants.MDS_APP_LABEL: 2,
-        constants.RGW_APP_LABEL: rgw_count,
         constants.NOOBAA_ENDPOINT_POD_LABEL: min_eps
     }
+    if not config.DEPLOYMENT['external_mode']:
+        resources_dict.update(
+            {
+                constants.MON_APP_LABEL: 3,
+                constants.CSI_CEPHFSPLUGIN_LABEL: number_of_worker_nodes,
+                constants.CSI_CEPHFSPLUGIN_PROVISIONER_LABEL: 2,
+                constants.CSI_RBDPLUGIN_LABEL: number_of_worker_nodes,
+                constants.CSI_RBDPLUGIN_PROVISIONER_LABEL: 2,
+                constants.OSD_APP_LABEL: osd_count,
+                constants.MGR_APP_LABEL: 1,
+                constants.MDS_APP_LABEL: 2,
+            }
+        )
+    if (
+            config.ENV_DATA.get('platform') in constants.ON_PREM_PLATFORMS
+            and not config.DEPLOYMENT['external_mode']
+    ):
+        # Workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1857802 - RGW count is 1
+        # post upgrade to OCS 4.5. Tracked with
+        # https://github.com/red-hat-storage/ocs-ci/issues/2532
+        rgw_count = 2 if float(config.ENV_DATA['ocs_version']) >= 4.5 and not (
+            post_upgrade_verification
+        ) else 1
```

Comment:
should be 
```python
rgw_count = 2 if float(config.ENV_DATA['ocs_version']) >= 4.5 else 1
```

Following https://github.com/red-hat-storage/ocs-ci/pull/2667

Reply:
oops , looks like I messed it while resolving conflicts.

Reply:
done